frontend/Controls/SearchBox.qml

```diff
@@ -1,55 +1,90 @@
-import QtQuick 2.0
+import QtQuick 2.7
 import QtQuick.Layouts 1.3
 import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
 import "../Controls" as Controls
 
-Rectangle {
-    id: root
+Item {
+    id: searchBoxWithResults
+
+    property alias text: searchField.text
+    property alias placeholder: searchField.placeholderText
+
     anchors.verticalCenter: parent.Center
     Layout.fillWidth: true
     Layout.maximumWidth: 367
     Layout.minimumWidth: 367
     Layout.preferredWidth: 367
-    height: 44
-    color: "#3A3E46"
-    radius: 5
 
-    property alias text: searchField.text
-    property alias placeholder: searchField.placeholderText
-    signal buttonClicked
+    anchors.top: parent.top
+    anchors.left: parent.left
+    anchors.topMargin: 13
+
+    z: 50
 
     Rectangle {
-        id: searchFieldContainer
-        Layout.fillWidth: true
+        id: searchBoxId
+
+        width: parent.width
         height: 44
-        width: 367
-        color: "transparent"
-        anchors.top: parent.top
-        anchors.left: parent.left
         radius: 5
 
-        TextField {
-            id: searchField
-            height: parent.height
-            width: 220
-            y: 0
+        color: "#3A3E46"
+
+        signal buttonClicked
 
-            font.pointSize: 14
-            font.family: "Roboto"
-            placeholderText: placeholder
+        Rectangle {
+            id: searchFieldContainer
+            Layout.fillWidth: true
+            height: 44
+            width: 367
+            color: "transparent"
+            anchors.top: parent.top
             anchors.left: parent.left
-            anchors.leftMargin: 15
-            style: TextFieldStyle {
-                textColor: "#7B7D82"
-                placeholderTextColor: "#7B7D82"
-                background: Rectangle {
-                    color: "transparent"
+            radius: 5
+
+            TextField {
+                id: searchField
+                height: parent.height
+                width: 220
+                y: 0
+
+                font.pointSize: 14
+                font.family: "Roboto"
+                placeholderText: placeholder
+                anchors.left: parent.left
+                anchors.leftMargin: 15
+                style: TextFieldStyle {
```

Comment:
This is fine for now, but FYI this is old syntax now. It's done via a `delegate` in qtquick2. 
Check out `Controls/TextInput.qml` for a rough example

Reply:
Can you give me more insight? I can probably fix it, but I don't get it yet.

Reply:
Generally if you search for "Qt Quick 2 _thing_" you'll find the more modern docs.
https://doc.qt.io/qt-5.10/qml-qtquick-controls2-textfield.html

Like I say not to worry for now, more of an FYI - we've got other controls that'll need updating so we can do it all at the same time